pkg/daemon/ceph/client/filesystem.go

```diff
@@ -175,6 +193,28 @@ func IsMultiFSEnabled() bool {
 	return false
 }
 
+// SetPoolSize method sets the sizes for MetadataPool and dataPool
+func SetPoolSize(context *clusterd.Context, clusterName, fsName string, dataPool []*model.Pool, metadataPool *model.Pool) error {
+	// updating the MetaDataPool size
+	metadataPool = GenerateMetadataPoolName(fsName, metadataPool)
+	err := SetPoolProperty(context, clusterName, metadataPool.Name, "size", strconv.Itoa(int(metadataPool.ReplicatedConfig.Size)))
+	if err != nil {
+		return errors.Wrapf(err, "failed to update metadataPool replica for filesystem %q", fsName)
```

Comment:
I think errors here should give the user information about what is wrong and how badly it is wrong. I would suggest that these messages include both the replica count that Rook tried to apply but failed, and upon failure we should try to see what the current replica count is to also report so that users can decide how badly their data safety is currently different from what they desire.

Reply:
Ok, so in the existing error Information, I will add something like
`failed to update metadataPool replica for filesystem %q, rook tried to change the replicaCount  from %d to %d.`
How is this?.

Reply:
It can be more succinct `failed to update metadataPool replica for filesystem %q from %d to %d`